crates/pop-parachains/src/templates.rs

```diff
@@ -136,6 +136,21 @@ pub enum Parachain {
 		)
```

Comment:
Whilst technically out of scope, should the generic OZ template supporte versions include v2.0.1?

Reply:
You're right, I missed that. We need to improve this so we don't have to manually add it each time. We should implement something like `SupportedVersionsFrom`.